github/src/main/proto/spine_examples/pingh/github/rest/fragments.proto

```diff
@@ -140,3 +140,46 @@ message CommentsResponse {
     //
     repeated CommentFragment item = 1 [(validate) = true];
 }
+
+// The GitHub REST API response received when obtaining verification codes.
+message VerificationCodesFragment {
+
+    // Device verification code.
+    string device_code = 1 [(required) = true, json_name = "device_code"];
+
+    // User verification code.
+    string user_code = 2 [(required) = true, json_name = "user_code"];
+
+    // The URL where users need to enter their `user_code`.
```

Comment:
"The URL of the GitHub resource, where users will be entering the verification code in scope of the device login flow."
+ link to the documentation will be helpful.